src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -0,0 +1,305 @@
+
+  <!-- Variables that CloudTest.Helix.Targets cares about:
+       **********************************************************************************************************************************
+       Required Properties:
+       **********************************************************************************************************************************
+       HelixApiAccessKey            - GitHub API Access Key for sending jobs to Helix.  Get from https://helix.dot.net/UserProfile/Index/
+       HelixJobType                 - Job type formatting string, used for sorting and display of jobs.
+       HelixSource                  - Job Source formatting string, used for sorting and display of jobs.
+       TargetQueue                  - Queue to send Jobs to. (TODO: Need discoverability for users)
+       BuildMoniker                 - Identifying name for build when sending to Helix.  Used for labelling
+       CloudDropConnectionString    - Azure Storage account connection string used for payloads.
+       CloudResultsConnectionString - Azure Storage account connection string used for results.
+       ArchivesRoot                 - Relative path for work item payloads to use.  Blob Uris will be made relative to this.
+       
+       **********************************************************************************************************************************
+       Optional Properties:
+       **********************************************************************************************************************************
+       TargetsWindows    - Whether to use Unix or Windows-specific path creation.             Default: False
+       TestListFilename  - File name to use for test list sent to Helix.                      Default: TestList.json
+       OverwriteOnUpload - Whether to overwrite blobs if they already exist on upload.        Default: False
+       ContainerName     - Container name for uploaded files.                                 Default: build-<Random GUID>. 
+       IsOfficial        - Boolean flag for display on Mission Control.                       Default: False:
+       HelixApiEndpoint  - Endpoint to send jobs to.                                          Default: https://helix.dot.net/api/2016-06-28/jobs
+       CloudResultsReadTokenValidDays  - Days that result URLs produced will be readble       Default: 30
+       CloudResultsWriteTokenValidDays - Days that result URLs produced will be write         Default: 4
+       
+       **********************************************************************************************************************************
+       Required ITaskItems:
+       **********************************************************************************************************************************
+       HelixWorkItem         - Zip files to be uploaded as payloads. 
```

Comment:
It might be good to include a sample set of property/item proj file that sets up the required and shows what folks should and can do. So folks can just copy/paste that into their repo and fill out the properties that are needed.